src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.Gss.cs

```diff
@@ -0,0 +1,336 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+using size_t = System.IntPtr;
+
+internal static partial class Interop
+{
+    internal static partial class libgssapi
+    {
+        [DllImport(Interop.Libraries.SecurityNative)]
+        internal static extern Status GssReleaseBuffer(
```

Comment:
To clarify Stephen's comment: We're naming them with that prefix in the shim (e.g. NetSecurityNative_GssReleaseBuffer) and using the Entrypoint attribute here.  But since the C# already calls it as Interop.Type.Method you don't need to duplicate the prefix here.